components/ResultsPageComponents/DistanceTravelledMode/DistanceTravelledMode.jsx

```diff
@@ -0,0 +1,153 @@
+import { Text, Flex, Box } from "@chakra-ui/react";
+import WalkingMan from "../../../public/images/survey-intro-icons/walking-man.svg";
+import Car from "../../../public/images/survey-intro-icons/car.svg";
+import Carpool from "../../../public/images/survey-intro-icons/carpool.svg";
+import Motorcycle from "../../../public/images/survey-intro-icons/motorcycle.svg";
+import Train from "../../../public/images/survey-intro-icons/train.svg";
+import DistanceTravelledModeChart from "./DistanceTravelledModeChart";
+
+export default function DistanceTravelledMode({ distanceTravelledMode }) {
```

Comment:
I have a questions for @hqtan , in terms of naming conventions, should the name for props be different from the name of the component? React doesn't complain but I personally would be interested to know.

Reply:
Hi @OlaShabalina !

Good pick up! I think you're right, the prop names should be different, to avoid confusion 👍🏽 
Lemme fix that 🙇🏽

Reply:
Hi @OlaShabalina ,
I've renamed prop; can you give PR another review plz? 🙏🏽

Reply:
Thanks @hqtan , everything else looks good to me!